src/loans/loan_state.py

```diff
@@ -254,6 +253,10 @@ def process_events_to_loan_states(
             'debt': [{token: float(amount) for token, amount in loan.debt.items()} for loan in state.loan_entities.values()],
         }
     )
+
+    # TODO: Upload health ratios etc here, rn every protocol has their
```

Comment:
I do it during the liquidable debt computation for MF. The reason is that for both computations we need to fetch some params (collateral/debt factors, mints, decimals, liquidation params, ..). So I prefer to do it at once.

Reply:
Okay, I'll remove the todo